iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_receiver_data.hpp

```diff
@@ -25,7 +25,8 @@ namespace iox
 {
 namespace popo
 {
-struct ChunkReceiverData : public ChunkQueueData
+template <uint32_t MaxChunksPerReceiver, typename ChunkQueueDataType>
```

Comment:
The parameter MaxChunksPerReceiver controls how many chunks the user is allowed to hold. I.e. how many memory can be blocked by chunks that are in processing. For me this is not reflected in this name. Better would be something like MaxChunksAllowedToBeHeld

Reply:
Or compared to the sending side something like MaxChunksProcessedInParallel

Reply:
What about "MaxChunksHeldSimultaneously"?